### ISSUE TITLE
Raising sysctl values without lowering them

### ISSUE THREAD

**Issue by Jamstah** [openshift/cluster-node-tuning-operator#85]
We have a requirement to increase `vm.max_map_count` for running Elastic Search. In OpenShift 3.11, we created a daemonset that ran code to check the current value, and if it was too low, to increase it. This to us seemed safe and allowed us to play nice with others.

In OpenShift 4.2, we're now trying to achieve the same effect with a tuning operator, but I'm struggling based on the documentation to work out how to do it. If we create a new profile and make it high priority enough, we're worried that other competing profiles would be overridden and perhaps other applications wouldn't get the tuning settings they require.

Is there a path to do this? I couldn't find one in the documentation. What would you recommend?

**Comment by tim-lo** [comment]
Would using the builtin `exec` function be advisable? Could write a script that checks for the existing value of `vm.max_map_count` and conditionally increase it.

**Comment by jmencak** [comment]
Sorry for the delay. Do not the default ES profiles [openshift-control-plane-es](https://github.com/openshift/cluster-node-tuning-operator/blob/master/assets/tuned/default-cr-tuned.yaml#L68) and [openshift-node-es](https://github.com/openshift/cluster-node-tuning-operator/blob/master/assets/tuned/default-cr-tuned.yaml#L77) work for you? They both inherit the settings from the parent openshift profiles.

**Comment by Jamstah** [comment]
Hi,

Thanks for the reply. Yes, we did find that after I opened this item and are now using them for elastic search.

We still have one pod that needs to raise the same limit even higher, so still have a custom profile. At the moment we have used the same inheritance as the elastic search profile provided by default, but we are still concerned about conflicts in the cluster.

Any advice on how to handle this?
